docs/install.md

```diff
@@ -92,6 +93,20 @@ If you wish to install an alternative gadget image, you could use the following
 $ kubectl gadget deploy --image=ghcr.io/myfork/inspektor-gadget:tag
 ```
 
+### Using the bcc image
+
+You can use the bcc image of Inspektor Gadget by using the `--bcc` flag:
+
+```bash
+$ kubectl gadget deploy --bcc
+```
+
+Note that, it will use the latest version of Inspektor Gadget.
```

Comment:
I think it's a bit misleading. It doesn't use the latest version but the default image version. For instance, v0.22.0 will use the `-v0.22.0-bcc` image, v0.23.0 will use `-v0.23.0-bcc` and so on.